tests/markdown.test.js

```diff
@@ -0,0 +1,18 @@
+const Markdown = require('shared/naturalcrit/markdown.js');
+
+// FIXME: tests below are disabled, becaues marked.js uses replaceAll function,
+// which is only supported starting with node.js 15.0.0:
+// https://github.com/nodejs/node/blob/master/doc/changelogs/CHANGELOG_V15.md#v8-86---35415
+// Need to enable them back when we uplifit node.js requirements of the project
+
+test.skip('Escapes <script> tag', function() {
+	const source = '<script></script>';
+	const rendered = Markdown.render(source);
+	expect(rendered).toMatch('&lt;script&gt;&lt;/script&gt;');
+});
+
+test.skip('Processes the markdown within an HTML block if its just a class wrapper', function() {
```

Comment:
It would also be really handy to do visual snapshot comparisons. Even if the HTML output is the same between updates to Homebrewery, we may change the CSS in such a way that messes up people's brews. Do you want to take a crack at image-pixel snapshot testing? Once this is set up? Looks like there's a good npm package to do this using Jest:

https://www.npmjs.com/package/jest-image-snapshot

Reply:
> Do you want to take a crack at image-pixel snapshot testing after this PR is merged?

If time permits, I'm ok to continue working on those tests and expand the coverage we have (but apparently I appear here something like for a 1-2 months within a year in average?)